build.properties

```diff
@@ -13,5 +13,6 @@ ImmersiveEngineering_version=0.12-+
 OpenComputers_version=MC1.12.2-1.7.2.+
 Patchouli_version=1.0-+
 RedstoneFlux_version=1.12-2.1.+
+MatterOverdrive_version=2699215
```

Comment:
Consider reusing ICBM notation so we get an idea of which version is being that magic number